internal/machine/query.go

```diff
@@ -0,0 +1,27 @@
+package machine
+
+import (
+	"context"
+
+	"github.com/samber/lo"
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/flaps"
+)
+
```

Comment:
The idea would be to keep the flaps package endpoint specific and any modifications or enhancements that outside of the flaps scope could be handled here.